### PR TITLE
Add reducer tests for Facility Locator

### PR DESCRIPTION
**Pull request by bshyong**
Resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/2686

### CODE REVIEW

File: src/js/facility-locator/reducers/facilities.js [hunk 1]
```diff
@@ -1,5 +1,3 @@
-import { FETCH_VA_FACILITY, FETCH_VA_FACILITIES, SEARCH_FAILED } from '../actions';
```
Comment (U-DON) [inline src/js/facility-locator/reducers/facilities.js]: Are these constants not going to be used from the `reducers/index` module anymore? If not, might wanna remove the lines where they're defined.

Comment (U-DON) [inline src/js/facility-locator/reducers/facilities.js]: In theory, they'd be useful to share between the usage here and the tests, but I don't anticipate much scaling for FL, so I'm not opposed to using hard-coded strings in two places.

File: test/facility-locator/reducers/searchQuery.unit.spec.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+    expect(state.error).to.eql(false);
+    expect(state.inProgress).to.eql(true);
+  });
+
+  it('should handle fetching list of facilities', () => {
+    const state = searchQueryReducer({
+      ...INITIAL_STATE,
+      inProgress: true,
+      error: true,
```
Comment (U-DON) [inline test/facility-locator/reducers/searchQuery.unit.spec.js]: Note there's no preset `error` in `INITIAL_STATE`.

File: test/facility-locator/reducers/searchQuery.unit.spec.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+      type: 'SEARCH_STARTED',
+    });
+
+    expect(state.error).to.eql(false);
+    expect(state.inProgress).to.eql(true);
+  });
+
+  it('should handle fetching list of facilities', () => {
+    const state = searchQueryReducer({
+      ...INITIAL_STATE,
```
Comment (U-DON) [inline test/facility-locator/reducers/searchQuery.unit.spec.js]: `INITIAL_STATE` is destructured here but not in the subsequent tests. Either add it in later tests or remove this one for consistency?

**Review by U-DON — APPROVED** [review]
LGTM with some nits.

File: src/js/facility-locator/reducers/facilities.js (hunk 1 above)
Comment (bshyong) [inline src/js/facility-locator/reducers/facilities.js]: There are some strange errors from Babel happening, when I include these so I removed their usage. I spent a bunch of time debugging this to no avail; we can remove these to your point about scaling. I'll remove the usage in the `reducers/index`

**Pull request merged**